pulpcore/cli/common/debug.py

```diff
@@ -53,7 +53,7 @@ def openapi_group() -> None:
 
 @openapi_group.command()
 @pass_pulp_context
-def schema(pulp_ctx: PulpContext) -> None:
+def spec(pulp_ctx: PulpContext) -> None:
```

Comment:
I am wondering if we should move the whole "openapi" command group one level up (from `pulp debug openapi` to `pulp openapi`).

Reply:
Hrm, I don't have a strong opinion either way. I think I lean ever so slightly to "leave it under the 'debug' top level, because that's what we're doing"